ui/app/common/pnc-client/resources/ProductVersionResource.js

```diff
@@ -34,10 +34,6 @@
         id: '@id'
       },
       {
-        query: {
```

Comment:
Why is it necessary to remove this method?

Reply:
@alexcreasy I thought this is for the endpoint: "GET /product-versions" which removed from the backend(as we discussed in NCL-5667"), but let me know if that is not for this endpoint or it should remain there for redundancy reason. Thanks, Alex!

Reply:
Yes, good point, I'd forgotten that! :)